code/game/machinery/computer/_computer.dm

```diff
@@ -29,6 +31,17 @@
 	. = ..()
 	power_change()
 
+	if(connectable && icon_state != "computer")
+		connectable = FALSE
+
+	for(var/obj/machinery/computer/selected in range(1,src))
+		addtimer(CALLBACK(selected, TYPE_PROC_REF(/atom, update_appearance)), 0.02 SECONDS)
+
+/obj/machinery/computer/Destroy()
+	for(var/obj/machinery/computer/selected in range(1,src))
+		addtimer(CALLBACK(selected, TYPE_PROC_REF(/atom, update_appearance)), 0.02 SECONDS)
+	return ..()
+
```

Comment:
niye timer ve 0.02 saniye sonra?

Reply:
procu direkt yapınca nedense olmadı, portladıgım yerde bu sekil cozum bulmus

Reply:
0.02 saniye destoryun bitmesi içindir. destory bitmeden update_appearance çağrılınca çağırılan computer sağına ve soluna bakarken gördüğü computerin silinmekte olup olmadığını bilmeyeceğinden. update_appearance 'da sağına soluna bakarken QDESTROYING() check ekle düzgün olur muhtemelen

Reply:
veya garanti için QDELETED kullanabilirsin

Reply:
beceremedim dedigini

Reply:
skill issue gibi

Reply:
tamam çözdüm commit atıcam şimdi